nexus-restore-p2/src/main/java/org/sonatype/nexus/blobstore/restore/p2/internal/P2RestoreBlobStrategy.java

```diff
@@ -91,7 +90,7 @@ protected boolean assetExists(@Nonnull final P2RestoreBlobData p2RestoreBlobData
   @Nonnull
   @Override
   protected List<HashAlgorithm> getHashAlgorithms() {
-    return ImmutableList.of(SHA1);
+    return HashAlgorithm.ALL_HASH_ALGORITHMS.values().stream().collect(Collectors.toList());
```

Comment:
Seems like `P2FacetImpl` still expects only SHA1

Reply:
> Seems like `P2FacetImpl` still expects only SHA1

Yea, I had switched to all hashes when I thought we would expire all p2 proxied assets but ended up reverting the main part of the change but missed the test.